src/components/BreadcrumbGroup/index.tsx

```diff
@@ -81,18 +73,22 @@ const BreadcrumbGroup = ({ items, rootPath = 'Home', availableRoutes = [] }: Bre
                                 const last = index === pathnames.length - 1;
                                 const segment = pathnames.slice(0, index + 1);
                                 const to = segment.length == 1 ? '/' : `/${segment.slice(1).join('/')}`;
+                                const text = value
```

Comment:
It is better to use a flag (e.g. disableCapitalizeFirstLetter) to support backward compatibility

Reply:
Discussed offline - better to preserve the casing of the hyperlinks as default behaviour :)